docs/tutorials/connecting-to-the-internet-via-usb.md

```diff
@@ -137,447 +108,216 @@ If you are using an older version, please upgrade.
             
             Debian GNU/Linux comes with ABSOLUTELY NO WARRANTY, to the extent
             permitted by applicable law.
-            root@ev3dev:~# 
+            robot@ev3dev:~$ 
+
+*   {: tab="Ubuntu"}<span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
```

Comment:
We should probably make an include template that takes an icon type and generates this icon lone so that we can change it in the future... Not important though